app/models/manageiq/providers/terraform_enterprise/automation_manager.rb

```diff
@@ -1,6 +1,8 @@
 class ManageIQ::Providers::TerraformEnterprise::AutomationManager < ManageIQ::Providers::ExternalAutomationManager
   supports :create
 
+  has_many :configuration_script_payloads, :foreign_key => :manager_id
```

Comment:
NOTE the base AutomationManager has this association as through configuration_script_sources, but since not all Workspaces are backed by git I made this a direct association with the manager (the table already has the manager_id)